ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e-ppc64le.yaml

```diff
@@ -0,0 +1,731 @@
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp/shared
+      command:
+      - cp
+      - /usr/bin/openshift-tests
+      - /usr/bin/oc
+      - /usr/bin/kubectl
+      - /tmp/shared/
+    containers:
+    # NOTE openshift-tests requires access to openstackclient/cinder
+    # so we re-use the install image here which already contains
+    # the required binaries. The initContainer above is used to copy over
+    # the latest openshift-tests/oc binaries.
+    #
+    # Once the cluster is up, executes shared tests
+    - name: test
+      image: registry.svc.ci.openshift.org/ppc64le-openstack-installer-43/ppc64le-ci:latest
+    # image: ${IMAGE_OPENSTACK_INSTALLER}
+      terminationMessagePolicy: FallbackToLogsOnError
+      resources:
+        requests:
+          cpu: 3
+          memory: 600Mi
+        limits:
+          memory: 4Gi
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp/shared
+      - name: cluster-profile
+        mountPath: /tmp/cluster
+      - name: artifacts
+        mountPath: /tmp/artifacts
+      env:
+      - name: ARTIFACT_DIR
+        value: /tmp/artifacts
+      - name: HOME
+        value: /tmp/home
+      - name: KUBECONFIG
+        value: /tmp/artifacts/installer/auth/kubeconfig
+      command:
+      - /bin/bash
+      - -c
+      - |
+        #!/bin/bash
+        set -euo pipefail
+
+        export PATH=/tmp/shared:/usr/libexec/origin:$PATH
+
+        trap 'touch /tmp/shared/exit' EXIT
+        trap 'jobs -p | xargs -r kill || true; exit 0' TERM
+
+        mkdir -p "${HOME}"
+
+        # wait for the API to come up
+        while true; do
+          if [[ -f /tmp/shared/exit ]]; then
+            echo "Another process exited" 2>&1
+            exit 1
+          fi
+          if [[ ! -f /tmp/shared/setup-success ]]; then
+            sleep 15 & wait
+            continue
+          fi
+          # don't let clients impact the global kubeconfig
+          cp "${KUBECONFIG}" /tmp/admin.kubeconfig
+          export KUBECONFIG=/tmp/admin.kubeconfig
+          break
+        done
+
+        # if the cluster profile included an insights secret, install it to the cluster to
+        # report support data from the support-operator
+        if [[ -f /tmp/cluster/insights-live.yaml ]]; then
+          oc create -f /tmp/cluster/insights-live.yaml || true
+        fi
+
+        # set up env vars
+        export KUBE_SSH_BASTION="$( oc --insecure-skip-tls-verify get node -l node-role.kubernetes.io/master -o 'jsonpath={.items[0].status.addresses[?(@.type=="ExternalIP")].address}' ):22"
+        export KUBE_SSH_KEY_PATH=/tmp/cluster/ssh-privatekey
+        export KUBE_SSH_USER=core
+        mkdir -p ~/.ssh
+        cp /tmp/cluster/ssh-privatekey ~/.ssh/kube_openstack_rsa || true
+
+        export OPENSHIFT_INSTALL_OS_IMAGE_OVERRIDE=https://art-rhcos-ci.s3.amazonaws.com/releases/rhcos-4.3-ppc64le/43.81.202001081114.0/ppc64le/rhcos-43.81.202001081114.0-openstack.ppc64le.qcow2.gz
```

Comment:
Why is this needed? Shouldn't https://github.com/openshift/installer/pull/3233 have provided this information?

Reply:
This patch was started long before that commit.  Let me try it without the override...

Reply:
Nope, still need the override.  I am seeing:
`level=info msg="Obtaining RHCOS image file from 'https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/rhcos-4.3/43.81.202003111353.0/x86_64/rhcos-43.81.202003111353.0-openstack.x86_64.qcow2.gz?sha256=4d204e638d365d9de121f5d513cff2567abd9232710f4bb79992efa4ba718008'"
`

Reply:
@hamzy does this mean there is a problem with the baked in rhcos image?
Maybe @Prashanth684 can help?

Reply:
Keep in mind that the CI pod runs on amd64 while the OpenShift cluster is a ppc64le cluster.

Reply:
is `architecture` specified in the install-config.yaml for the master and computes? if not it's going to pick x86

Reply:
Thanks Prashanth for pointing out the needed architecture keys in install-config.yaml